src_c/color.c

```diff
@@ -298,6 +304,18 @@ _get_double(PyObject *obj, double *val)
     return 1;
 }
 
+static int
+_get_int(PyObject *obj, int *val)
+{
+    PyObject *intobj;
+    if (!(intobj = PyNumber_Long(obj))) {
+        return 0;
+    }
+    *val = PyLong_AsLong(intobj);
+    Py_DECREF(intobj);
```

Comment:
Is there a reason for doing this over just a `*val = PyLong_AsLong(obj)`?
You could just do 
```C
static int
_get_int(PyObject *obj, int *val) {
    *val = PyLong_AsLong(obj);
    if (PyErr_Occurred())
        return 0;
    return 1;
}
```
Or possibly better 
```C
static int
_get_int(PyObject *obj, int *val) {
    if (PyFloat_Check(obj)) {
        *val = (int)PyFloat_AS_DOUBLE(obj);
        return 1;
    }
    *val = PyLong_AsLong(obj);
    if (PyErr_Occurred())
        return 0;
    return 1;
}
```